requirements-cpu.txt

```diff
@@ -2,6 +2,6 @@
 -r requirements-common.txt
 
 # Dependencies for x86_64 CPUs
-torch == 2.3.1+cpu; platform_machine != "ppc64le"
-torchvision == 0.18.1+cpu; platform_machine != "ppc64le"   # required for the image processor of phi3v, this must be updated alongside torch
+torch == 2.4.0; platform_machine != "ppc64le"
```

Comment:
Hi @bigPYJ1151 and @WoosukKwon , this change missed the suffix `+cpu` for torch version which leads to the build failure of the CPU target.
Please take a look at https://github.com/vllm-project/vllm/pull/6931 .
Thanks.